lldb/source/Core/PluginManager.cpp

```diff
@@ -209,7 +209,7 @@ template <typename Instance> class PluginInstances {
     Instance instance =
         Instance(name, description, callback, std::forward<Args>(args)...);
     m_instances.push_back(instance);
```

Comment:
While you're here... :-) Should this use `emplace_back` to create the instance object in place?